CHANGES/534.doc

```diff
@@ -0,0 +1,2 @@
+Added steps to remove the plugin and announced its EOL.
+Pulp-2to3-migration plugin reaches its EOL on XXX, the last supported pulpcore version is 3.19.
```

Comment:
https://discourse.pulpproject.org/t/pulp-2to3-migration-plugin-end-of-life-eol-date/457

Reply:
@ggainey , could you weigh in here or on the thread please?